deletionwatcher.py

```diff
@@ -44,23 +44,23 @@ def check_websocket_for_deletion(self, post_site_id, post_url, timeout):
             try:
                 a = ws.recv()
             except websocket.WebSocketTimeoutException:
-                t_metasmoke = Thread(target=Metasmoke.send_deletion_stats_for_post,
-                                     args=(post_url, False))
+                t_metasmoke = Thread(target=Metasmoke.send_deletion_stats_for_post, args=(post_url, False))
                 t_metasmoke.start()
                 return False
             if a is not None and a != "":
                 try:
                     d = json.loads(json.loads(a)["data"])
                 except:
                     continue
-                if d["a"] == "post-deleted" and str(d["qId"]) == question_id and ((post_type == "answer" and "aId" in d and str(d["aId"]) == post_id) or post_type == "question"):
-                    t_metasmoke = Thread(target=Metasmoke.send_deletion_stats_for_post,
-                                         args=(post_url, True))
+                if d["a"] == "post-deleted" \
```

Comment:
Probably good to split this one over two lines, but you can probably make line 56 part of line 55 and still keep it readable